test/test_torch.py

```diff
@@ -17608,9 +17595,11 @@ def test_split_view(self, device):
 
 _complex_types = [torch.cfloat, torch.cdouble]
 
-_float_types_no_half = [torch.float, torch.double]
+_complex_and_types2 = _complex_types + _types2
```

Comment:
Could you refactor these to use `torch.testing.get_all_XXX_dtypes()` functions that have a very similar purpose?